src/components/MonacoEditor.tsx

```
@@ -0,0 +1,15 @@
+import * as monaco from 'monaco-editor'
+import { loader } from '@monaco-editor/react'
+
+loader.config({ monaco })
+
+interface Props {
+    value: string
+    onChange: (value: string) => void
+    language?: string
+}
+
+export default function MonacoEditor({ value, onChange, language = 'shell' }: Props) {
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)